### PR TITLE
Use stomp-1.1 if possible.

### PR DESCRIPTION
**Pull request by ralphbean**
This depends on oisinmulvihill/stomper#7 being reviewed, merged, and released.
If that gets rejected then this probably shouldn't be merged.

### CODE REVIEW

File: moksha.hub/moksha/hub/stomp/protocol.py [hunk 1]
```diff
@@ -65,13 +73,18 @@ def ack(self, msg):
 
     def subscribe(self, dest, **headers):
         f = stomper.Frame()
-        f.unpack(stomper.subscribe(dest))
+        if stomper.STOMP_VERSION != '1.0':
+            f.unpack(stomper.subscribe(dest, dest))
```
Comment (lmacken) [inline moksha.hub/moksha/hub/stomp/protocol.py:86]: So, stomp 1.0 doesn't need this anymore?

Comment (ralphbean) [inline moksha.hub/moksha/hub/stomp/protocol.py:86]: Ooop, bug.  thanks!

**Comment by lmacken** [comment]
:+1:

**Comment by lmacken** [comment]
Latest changes look good to me, :thumbsup:

**Comment by ralphbean** [comment]
OK, it's all merged upstream and packaged for Fedora/EPEL now.

**Pull request merged**